src/utils.ts

```diff
@@ -32,3 +36,15 @@ export function verifyContainerProivder(containerProvider: string): 'wsl' | 'hyp
     return undefined;
   }
 }
+
+export async function pullImageFromRedHatRegistry(imageSha: string, pathToSave: string): Promise<void> {
+  const redirectToImage = await rhsmClientV1?.images?.downloadImageUsingSha(imageSha);
```

Comment:
You should get `rhsmClientV1` as parameter (of type `SubscriptionManagerClientV1`), instead of relying on the one defined in another file